test.js

```diff
@@ -0,0 +1,9 @@
+class FooTest {
+	test() {
+		const c = 8;
+		let a = [];
+		for (let b of a) {
+			console.log('foo');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)